wiki/People/The_Team/Support_Team/ru.md

```diff
@@ -1,37 +1,25 @@
----
-outdated: true
----
+# Команда поддержки
 
-# Техподдержка
+**Команда поддержки** (иногда **Support Team Redux**) — группа пользователей, модерирующих следующие форумы: [Development](https://osu.ppy.sh/community/forums/2), [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) (за исключением [Tournaments](https://osu.ppy.sh/community/forums/55) и [Mapping Techniques](https://osu.ppy.sh/community/forums/61)), [Skinning](https://osu.ppy.sh/community/forums/15), [Feature Requests](https://osu.ppy.sh/community/forums/4) и [Help](https://osu.ppy.sh/community/forums/5).
 
-**Техподдержка** (Support Team Redux, «Техподдержка Возрождённая» — [Ephemeral](https://osu.ppy.sh/users/Ephemeral), 2014) — группа, участники которой следят за следующими форумами:
+Команду поддержки **не стоит путать** со службой поддержки (или просто техподдержкой), которая решает вопросы, связанные с аккаунтами и рестриктами, и отвечает по адресам [support@ppy.sh](mailto:support@ppy.sh) и [accounts@ppy.sh](mailto:support@ppy.sh).
```

Comment:
this paragraph isn't present in English, is it something only appropriate for a translation?

Reply:
not really, just the wording that's more clear and straightforward. I'll make the same change to the original soon